### PR TITLE
No longer include --scan when displaying the build command

### PR DESCRIPTION
**Pull request by jthurne**
#### Before
image

image

#### After
image

image

### CODE REVIEW

File: components/scripts/gradle/01-validate-incremental-building.sh [hunk 1]
```diff
@@ -149,12 +149,10 @@ execute_build() {
 
 print_gradle_command() {
   local scan_arg
-  if [[ "${build_scan_publishing_mode}" == "on" ]]; then
-    scan_arg="--scan"
-  else
-    scan_arg="-Dscan.dump"
+  if [[ "${build_scan_publishing_mode}" != "on" ]]; then
```
Comment (erichaagdev) [inline components/scripts/gradle/01-validate-incremental-building.sh:152]: It looks like we typically check for `on` vs `off` in the codebase.

```suggestion
  if [[ "${build_scan_publishing_mode}" == "off" ]]; then
```

File: components/scripts/gradle/01-validate-incremental-building.sh [hunk 2]
```diff
@@ -149,12 +149,10 @@ execute_build() {
...
+    scan_arg=" -Dscan.dump"
   fi
-  info "./gradlew --no-build-cache ${scan_arg} -Dscan.tag.${EXP_SCAN_TAG} -Dscan.value.runId=${RUN_ID} $*$(print_extra_args)"
+  info "./gradlew --no-build-cache${scan_arg} -Dscan.tag.${EXP_SCAN_TAG} -Dscan.value.runId=${RUN_ID} $*$(print_extra_args)"
```
Comment (erichaagdev) [inline components/scripts/gradle/01-validate-incremental-building.sh:155]:
```suggestion
  info "./gradlew --no-build-cache ${scan_arg} -Dscan.tag.${EXP_SCAN_TAG} -Dscan.value.runId=${RUN_ID} $*$(print_extra_args)"
```

Comment (erichaagdev) [reply components/scripts/gradle/01-validate-incremental-building.sh:155]: This was intentional.

File: components/scripts/gradle/01-validate-incremental-building.sh [hunk 3]
```diff
@@ -149,12 +149,10 @@ execute_build() {
 
 print_gradle_command() {
```
Comment (etiennestuder) [inline components/scripts/gradle/01-validate-incremental-building.sh:150]: I suggest to use the same if/else construct with the almost-duplicated command as in the other experiments.

File: components/scripts/gradle/01-validate-incremental-building.sh [hunk 4]
```diff
@@ -149,12 +149,10 @@ execute_build() {
...
   local scan_arg
```
Comment (etiennestuder) [inline components/scripts/gradle/01-validate-incremental-building.sh:151]: I suggest to use the same if/else construct with the almost-duplicated command as in the other experiments.

**Review by etiennestuder — CHANGES_REQUESTED** [review]

File: components/scripts/gradle/01-validate-incremental-building.sh (hunk 4 above)
Comment (jthurne) [reply components/scripts/gradle/01-validate-incremental-building.sh]: Great idea. ✅ Done.

**Review by etiennestuder — APPROVED** [review]

**Review by erichaagdev — APPROVED** [review]

**Pull request merged**